modules/nf-core/kraken2/add/main.nf

```diff
@@ -0,0 +1,38 @@
+process KRAKEN2_ADD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-5799ab18b5fc681e75923b2450abaa969907ec98:87fc08d11968d081f3e8a37131c1f1f6715b6542-0' :
+        'biocontainers/mulled-v2-5799ab18b5fc681e75923b2450abaa969907ec98:87fc08d11968d081f3e8a37131c1f1f6715b6542-0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path taxonomy_names
+    path taxonomy_nodes
+    path accession2taxid
+
+    output:
+    tuple val(meta), path("$prefix/"), emit: db
```

Comment:
```suggestion
    tuple val(meta), path("$prefix"), emit: db
```

I seem to remember you can't specify directories with `/` but lets see... this comment suggestion is optional :)